src/table/sticky-scrollbar/__tests__/sticky-scrollbar.test.tsx

```diff
@@ -14,7 +14,7 @@ it('adds an offset if the scrollbar is an overlay', () => {
   const tableRef = { current: document.createElement('table') };
   const wrapperRef = { current: document.createElement('div') };
 
-  browserScrollbarSizeMock.mockImplementationOnce(() => ({ height: 0, width: 100 }));
+  browserScrollbarSizeMock.mockImplementation(() => ({ height: 0, width: 100 }));
```

Comment:
Replaced mockImplementationOnce by mockImplementation because StickyScrollbar component uses useStickyScrollbar under the hood, which in turn conditionally calls browserScrollbarSize() if supportsStickyPosition() === true. However these test suites previously implied that supportsStickyPosition always returns false. So mockImplementationOnce returns the valid mock result only once, all further calls ended up with undefined